pkg/driver/controller.go

```diff
@@ -201,6 +204,13 @@ func (d *controllerService) CreateVolume(ctx context.Context, req *csi.CreateVol
 		return newCreateVolumeResponse(disk), nil
 	}
 
+	// check if a request is already in-flight because the CreateVolume API is not idempotent
+	if ok := d.inFlight.Insert(req); !ok {
+		msg := fmt.Sprintf("Create volume request for %s is already in progress", volName)
+		return nil, status.Error(codes.AlreadyExists, msg)
```

Comment:
CSI spec says use "ABORTED" for an operation in progress: https://github.com/container-storage-interface/spec/blame/master/spec.md#L470